internal/endtoend/testdata/pg_advisory_xact_lock/go/query.sql.go

```diff
@@ -11,8 +11,8 @@ const advisoryLock = `-- name: AdvisoryLock :many
 SELECT pg_advisory_unlock($1)
 `
 
-func (q *Queries) AdvisoryLock(ctx context.Context, key int64) ([]bool, error) {
-	rows, err := q.db.QueryContext(ctx, advisoryLock, key)
+func (q *Queries) AdvisoryLock(ctx context.Context, pgAdvisoryUnlock int64) ([]bool, error) {
```

Comment:
While the PostgreSQL documentation claims the first parameter to `pg_advisory_unlock` is named `key`, that name isn't actually included in the definition of the function stored in the database.